src/connectors/xiami.js

```diff
@@ -1,22 +1,19 @@
 'use strict';
 
-Connector.playerSelector = '#player-main';
+Connector.playerSelector = '.play-bar';
 
-Connector.trackSelector = '#J_trackName';
+Connector.trackSelector = '.music .info .title';
 
-Connector.albumSelector = '.ui-track-current .c3>a';
+Connector.artistSelector = '.music .info .singers';
 
-Connector.artistSelector = '#J_trackInfo>a:nth-child(2)';
-
-Connector.playButtonSelector = '.play-btn';
-
-Connector.currentTimeSelector = '#J_positionTime';
-
-Connector.trackArtSelector = '#J_playerCoverImg';
+Connector.trackArtSelector = '.music .cover-link .active img';
 
 Connector.getDuration = () => {
-	let text = Connector.getCurrentTime() ? $('#J_durationTime').text() : '';
-	return Util.stringToSeconds(text);
+	let text = $('.audio-progress .range .bar .handle').val();
+	let str = text.split('/');
+	return str[0];
```

Comment:
Is this correct? Isn't `0` the currentTime not the duration? Could we cover currentTime as well as I saw it was removed.

Reply:
When the page's music is playing, currentTimes is viewed like 01:03/04:12.
 So i get currentTime using 
`let str = text.split('/');`
`return str[0];`
If it don't be needed, I'll erase it and commit it again.

Reply:
I still think this is wrong. so text will contain `00:30/02:45` for example and splitting this will yeild `00:30` in position `0`... which is the current time - not the duration. Does that make sense?

Perhaps a better solution would be to just set

```
Connector.timeInfoSelector = '.audio-progress .range .bar .handle';
```

This handles the case when the duration and remaining time is in this format.

Reply:
> I still think this is wrong. so text will contain `00:30/02:45` for example and splitting this will yeild `00:30` in position `0`... which is the current time - not the duration. Does that make sense?
> 
> Perhaps a better solution would be to just set
> 
> ```
> Connector.timeInfoSelector = '.audio-progress .range .bar .handle';
> ```
> This handles the case when the duration and remaining time is in this format.

Oh.. I understand!
I modified duration part just now.
Is it right?